src/app.js

```diff
@@ -0,0 +1,120 @@
+"use strict";
+//Someandsome
+function createApp() {
+  return new (class {
+    constructor() {
+      this.chitterApi = new chitterApi();
+      this.createViews = new CreateViews();
+      this.allPeeps = this.callPeeps();
+      this.owner = "";
+      this.userId = "";
+      this.sessionKey = "";
```

Comment:
Do you definitely need to store the user information as instance variables in this class? Or could you use session/local storage (or something similar)? I genuinely don't know btw because I didn't get this far

Reply:
I also don't know which option is better/worse or if they're the same

Reply:
Normally the session would be maintained by the server via session cookies(even if the auth is api constructed) in this case using http-server which looks more like a testing server I thought I will not maintain this feature.